core/src/main/java/org/jobrunr/jobs/details/instructions/BAStoreOperandInstruction.java

```diff
@@ -14,7 +15,7 @@ public void load() {
     }
 
     @Override
-    public Object invokeInstruction() {
+    public @Nullable Object invokeInstruction() {
         return null;
```

Comment:
At least some of the implementations here use `return null`; would it make sense to pull up the annotation to the super class or interface for this method?